sdk/tables/data-tables/src/TableServiceClient.ts

```diff
@@ -182,6 +188,12 @@ export class TableServiceClient {
     this.pipeline = client.pipeline;
     this.table = client.table;
     this.service = client.service;
+
+    const secondaryUrl = getSecondaryUrlFromPrimarystri(this.url);
+    this.secondaryServiceOperations = new GeneratedClient(secondaryUrl, {
```

Comment:
rather than incurring the overhead of a second client, I think we could solve this in a couple other ways:

1. Set a custom header when making the getStatistics request and add a custom pipeline policy that sees this header, removes it, and updates the host to the secondary host address
2. Override the baseUri of the operationSpec for getStatistics to be `{url}` and set the `url` property on the generated client to be the secondary host name since we don't seem to use it anywhere else (though you could also invent a new property if you wanted)

Reply:
Yeah, that's a good idea. I'm going with option 1)